samples/client/petstore/php/OpenAPIClient-php/docs/Model/Capitalization.md

```diff
@@ -3,12 +3,12 @@
 ## Properties
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**small_camel** | **string** |  | [optional] 
-**capital_camel** | **string** |  | [optional] 
-**small_snake** | **string** |  | [optional] 
-**capital_snake** | **string** |  | [optional] 
-**sca_eth_flow_points** | **string** |  | [optional] 
-**att_name** | **string** | Name of the pet | [optional] 
+**small_camel** | **object** |  | [optional] 
```

Comment:
@fMads looks like something wrong. Not related to change in this PR. All type becomes "object".

Can you rebase this PR on the latest master (or merge the latest master into this PR), build the JAR (mvn clean package) and try to update the PHP samples again?

Reply:
Yes - done :)